tests/meltano/cli/test_initialize.py

```diff
@@ -71,7 +71,7 @@ def test_init_existing_empty_directory(self, cli_runner, tmp_path_factory, pushd
         project_dir.mkdir()
         assert project_dir.exists()
 
-        result = cli_runner.invoke(cli, ["init", "test_project", "--no_usage_stats"])
+        result = cli_runner.invoke(cli, ["init", "test_project", "--no_usage-stats"])
```

Comment:
The mix of hyphens and underscores here is intentionally. Additionally, some tests are unchanged, and continue to use `--no_usage_stats` instead of `--no-usage-stats`, which ensures that the old option names still work.